src/vcSettingsUI.cpp

```diff
@@ -368,7 +370,13 @@ void vcSettingsUI_Show(vcState *pProgramState)
           }
 
           if (pProgramState->settings.visualization.mode == vcVM_Displacement)
-            ImGui::InputFloat2(vcString::Get("settingsVisDisplacementRange"), &pProgramState->settings.visualization.displacement.x);
+          {
+            if (ImGui::InputFloat2(vcString::Get("settingsVisDisplacementRange"), &pProgramState->settings.visualization.displacement.x))
```

Comment:
From memory, this won't trigger if someone uses Ctrl+Click to enter the value?

Reply:
these aren't sliders, just input floats

Reply:
Oh right, sorry.